goss.yaml

```diff
@@ -20,3 +13,24 @@ file:
     mode: "0755"
     owner: root
     group: root
+
+port:
+  tcp6:1777:
+    listening: true
+    port: 'tcp6:1777'
+    skip: false
+
+  tcp6:55679:
+    listening: true
+    port: 'tcp6:55679'
+    skip: false
+
+  tcp6:4317:
+    listening: true
+    port: 'tcp6:4317'
+    skip: false
+
+  tcp6:4318:
+    listening: true
+    port: 'tcp6:4318'
+    skip: false
```

Comment:
- Should we add a comment here or elsewhere about the purpose of each port?
- Should we have [tcp, tcp6 or both](https://goss.readthedocs.io/en/stable/gossfile/?h=tcp6#port)?

Reply:
I need to better understand how otelcol works... so far I've discovered that all the open ports are tcp6.

For now I'll test these:

```yaml
port:
  # otlp receiver grpc
  tcp6:4317:
    listening: true
    port: 'tcp6:4317'
    skip: false

  # otlp receiver http
  tcp6:4318:
    listening: true
    port: 'tcp6:4318'
    skip: false

  # prometheus receiver http
  tcp6:8888:
    listening: true
    port: 'tcp6:8888'
    skip: false
```